src/main/java/com/greeenai/greeenai/global/config/WebClientConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.greeenai.greeenai.global.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.ExchangeStrategies;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class WebClientConfig {
+
+    @Bean
+    public WebClient kaggomWebClient(WebClient.Builder builder) {
+        return builder.baseUrl("https://kaggom.online")
+                .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
                .defaultHeader(CONTENT_TYPE, APPLICATION_JSON_VALUE)
```

static import 바랍니다.
아래에 있는 것도 같이 처리 부탁드려요.